### PR TITLE
Split website_sale_search_custom into 2 modules

### PR DESCRIPTION
**Pull request by ilmir-k**
Split website_sale_search_custom into 2 modules

### CODE REVIEW

**Comment by yelizariev** [comment]
IOError: File not found: website_sale_search_tags/views.xml

File: website_sale_search_tags/views.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+  </data>
+</openerp>
```
Comment (yelizariev) [inline website_sale_search_tags/views.xml]: Can remove this from ``__openerp__.py`` instead of adding empty file

Comment (ilmir-k) [inline website_sale_search_tags/views.xml]: ok

**Comment by yelizariev** [comment]
update another module too

IOError: File not found: website_sale_search_custom/views.xml

**Comment by ilmir-k** [comment]
It's working on runbot

**Pull request merged**